locale/en/docs/guides/buffer-constructor-deprecation.md

```diff
@@ -31,10 +31,11 @@ If you’re using Node.js ≥ 8.0.0 (which is recommended), Node.js exposes mult
 - `--trace-deprecation` does the same thing, but only for deprecation warnings.
 - `--pending-deprecation` will show more types of deprecation warnings. In particular, it will show the `Buffer()` deprecation warning, even on Node.js 8.
 
-You can set these flags using an environment variable:
+You can set these flags using environment variables:
 
-```console
-$ export NODE_OPTIONS='--trace-warnings --pending-deprecation'
+```bash
+$ export NODE_OPTIONS=--trace-warnings
+$ export NODE_PENDING_DEPRECATION=1
```

Comment:
It seems [we have not](https://nodejs.org/download/nightly/v11.0.0-nightly20180530c1a87a30e7/docs/api/cli.html#cli_node_options_options) `--pending-deprecation` in `NODE_OPTIONS`.

Reply:
@vsemozhetbyt https://nodejs.org/download/nightly/v11.0.0-nightly20180530c1a87a30e7/docs/api/cli.html#cli_pending_deprecation

Reply:
Yes, we have it as a separate cli key, but not as a NODE_OPTIONS env variable part option.

Reply:
I'm not sure I follow, `--pending-deprecation` is a CLI option right? If so it should work when used via `NODE_OPTIONS` which allows to specify CLI options in the environment via a space-separated list.

Reply:
It seems we have [a whitelist of cli keys that are allowed in `NODE_OPTIONS`](https://nodejs.org/download/nightly/v11.0.0-nightly20180609ef1f130041/docs/api/cli.html#cli_node_options_options):

> Node options that are allowed are: ...

Reply:
Was it removed recently? It works on Node.js 10.

Reply:
I think it is simply not listed there.

Reply:
Node.js 10 doc has not it as well:

https://nodejs.org/dist/latest-v10.x/docs/api/cli.html#cli_node_options_options

If it does work, maybe the API docs need updating.

Reply:
Yes it seems outdated, see https://github.com/nodejs/node/blob/master/src/node.cc#L3156-L3198

Reply:
Thank you. I will revert this change.

Reply:
PR to update the API doc: https://github.com/nodejs/node/pull/21229